index.js

```diff
@@ -131,7 +133,67 @@ alexa.response = function(session) {
     this.sessionObject.clear(key);
     return this;
   };
-
+  this.audioPlayerPlay = function (url, token, playBehavior, offsetInMilliseconds, expectedPreviousToken) {
```

Comment:
I still think this is not future-proof. Every time Amazon adds something to audioplay we'll be throwing more parameters in here. What do you think about leaving this to the user and not being too smart about it?

```js
this.audioPlayerPlay = function (playBehavior, audioItem) {

}

this.audioPlayerPlayStream = function (playBehavior, stream) {
  
}
```